core-bundle/contao/languages/en/explain.xlf

```diff
@@ -86,6 +86,324 @@
+        <source>Photograph, icon, or other image corresponding to the company, person, address, or contact information in the other fields associated with this field</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.43.0">
+        <source>tel</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.43.1">
+        <source>Full telephone number, including country code</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.44.0">
+        <source>tel-country-code</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.44.1">
+        <source>Country code component of the telephone number</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.45.0">
+        <source>tel-national</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.45.1">
+        <source>Telephone number without the country code component, with a country-internal prefix applied if applicable</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.46.0">
+        <source>tel-area-code</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.46.1">
+        <source>Area code component of the telephone number, with a country-internal prefix applied if applicable</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.47.0">
+        <source>tel-local</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.47.1">
+        <source>Telephone number without the country code and area code components</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.48.0">
+        <source>tel-local-prefix</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.48.1">
+        <source>First part of the component of the telephone number that follows the area code, when that component is split into two components</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.49.0">
+        <source>tel-local-suffix</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.49.1">
+        <source>Second part of the component of the telephone number that follows the area code, when that component is split into two components</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.50.0">
+        <source>tel-extension</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.50.1">
+        <source>Telephone number internal extension code</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.51.0">
+        <source>email</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.51.1">
+        <source>E-mail address</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.52.0">
+        <source>impp</source>
+      </trans-unit>
+      <trans-unit id="XPL.autocomplete.52.1">
+        <source>URL representing an instant messaging protocol endpoint (for example, "aim:goim?screenname=example" or "xmpp:fred@example.net")</source>
+      </trans-unit>
```

Comment:
Nice 🙂. Should we also add `section-*`?

Reply:
@fritzmg see 147481c7396977f439cea01e7986cc6bcdd9a096

![image](https://github.com/user-attachments/assets/54700cea-354d-4adf-a4c0-0985ee51887c)